integrations/transportation/ryanair.js

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment';
+import request from 'superagent';
+import { ACTIVITY_TYPE_TRANSPORTATION, TRANSPORTATION_MODE_PLANE } from '../../definitions';
+
+const agent = request.agent();
+const BASE_URL = 'https://api.ryanair.com/userprofile/rest/api/v1/';
+const LOGIN_URL = `${BASE_URL}login`;
+const PROFILE_URL = `${BASE_URL}secure/users/`;
+// user info available at `${PROFILE_URL}${customerId}/profile/full/`
+
+async function logIn(username, password) {
+  const res = await agent
+    .post(LOGIN_URL)
+    .type('application/x-www-form-urlencoded')
+    .set('Authorization', 'Basic ')
+    .send({
+      password,
+      username,
+    });
+
+  if (!res.ok) {
+    console.error('--------RESPONSE---------', res);
```

Comment:
I think here you can just remove this line as it seems only to be relevant when developing the integration